sw/device/lib/runtime/otbn.c

```diff
@@ -195,3 +196,16 @@ otbn_result_t otbn_zero_data_memory(otbn_t *ctx) {
   }
   return retval;
 }
+
+void otbn_dump_dmem(const otbn_t *ctx, uint32_t max_addr) {
+  if (max_addr == 0) {
+    max_addr = dif_otbn_get_dmem_size_bytes(&ctx->dif);
+  }
+
+  uint32_t data[8];
+  for (int i = 0; i < max_addr / 32; ++i) {
```

Comment:
Wouldn't it make more sense to increment by += 32 and then divide by it in the LOG_INFO? I don't actually care but that's what I would have done.

Reply:
Done that.